### PR TITLE
Updates on header, README and bug fixes

### PR DESCRIPTION
**Pull request by caike08**
Hey buddy, 

I've found a bug in our last pull request, so I'm sending this to fix it. Next time I'll run it locally before merging! Here is what I've done:
- Updated meta tags on header, before it the user could zoom in our page on his phone and it wasn't a good thing for our responsivity, so I disabled it.
- Updated README.md, it was time! Now we have a more detailed readme file about our project. Feel free to update it if you want so.
- Bug fixes, now all tabs and menubar links work correctly.

### CODE REVIEW

File: client/html/pages/panel/settings/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
                 <!-- BEGIN CONTACTS NAV -->
                 <div class="col-sm-5 col-md-4 col-lg-3">
-                  <ul class="nav nav-pills nav-stacked nav-tab" data-toggle="tabs">
+                  <ul class="nav nav-pills nav-stacked nav-tabs-vert" data-toggle="tabs">
```
Comment (MarceloBorgesP) [inline client/html/pages/panel/settings/index.html:28]: Why do we need this class **nav-tabs-vert**? is it for css rules? if so, leave both **nav-tabs** and **nav-tabs-vert** (if possible) then we wont't need to create one more event handler

**Review by MarceloBorgesP — COMMENTED** [review]
No errors, only a suggestion :+1:

**Comment by MarceloBorgesP** [comment]
BTW, looks a lot better our read.me now :smile:

File: client/html/pages/panel/settings/index.html (hunk 1 above)
Comment (caike08) [reply client/html/pages/panel/settings/index.html:28]: This is how it looks like if I use "nav-tabs" to render it:
image

And this is what happens if I use the same structure as used [here](https://github.com/caike08/BancoTalentosTCC/blob/updates/client/html/pages/panel/jobs/index.html#L25):
image

The second one is almost there, but it adds up a border in the last child. So I'm using a different one just for when a panel is inside settings. Got it?

**Review by MarceloBorgesP — APPROVED** [review]
Great job!

**Pull request merged**